scripts/lighthouse_checks.py

```diff
@@ -0,0 +1,172 @@
+# Copyright 2020 The Oppia Authors. All Rights Reserved.
```

Comment:
Rename this file to `run_lighthouse_tests.py`.

Reply:
Done